tests/features/joinup_core/owner_deletion.feature

```diff
@@ -0,0 +1,63 @@
+@api
```

Comment:
Wouldn't it be better to add this in the existing feature that is dealing with deleting owners (`owner.moderation.feature`), or at least in the same folder?

Reply:
The 'owner' folder is about rdf entity type 'owner's. It is the corresponding entity to the contact_information. Not the actual user. It is a bit confusing..